hugo/layouts/partials/teaser--blog.html

```diff
@@ -0,0 +1,29 @@
+<article class="teaser teaser--blog">
+    {{ if .Params.Image }}
+        <div class="media">
+            <figure class="teaser__image">
+                <img src={{ .Params.Image }} alt=""/>
+            </figure>
+        </div>
+    {{ end }}
+
+    <div class="teaser__content">
+        <div class="teaser__heading">
+            <h2 class="teaser__title">{{ .Title }}</h2>
+            {{ if .Params.tags }}
+                {{ $tags := where .Site.Params.tags "name" "in"  .Params.tags }}
+                {{- partial "tags.html" (dict "listClass" "teaser__tags" "tags" $tags) -}}
+            {{ end }}
+        </div>
+        <p class="teaser__meta">{{ .Date.Format ($.Param "time_format") }}</p>
+        <p class="teaser__excerpt">{{ .Summary | truncate 195 }}</p>
+        <div class="link teaser__readmore">
+            <span class="link__text">{{ T "ui_read_more" }}</span>
+        </div>
+    
+        <a class="teaser__link" aria-label="{{ T "ui_read_more" }} - {{ .LinkTitle }}" href="{{ .RelPermalink }}">
+            <span>{{ T "ui_read_more" }}</span>
```

Comment:
aria-label and text itself are now the same. Won't that be a problem for screenreaders where the same text gets to be read twice?